components/ParallaxScroll.js

```diff
@@ -0,0 +1,24 @@
+export default class Parallax extends React.Component {
+  componentDidMount () {
+    window.addEventListener('load', this.scrollLoop, false)
+  }
+
+  componentWillUnMount () {
+    window.removeEventListener('load', this.scrollLoop, false)
+  }
+
+  setPosition = (yPos) => {
+    this.dom.style.transform = `translate3d(0, ${yPos}px, 0)`
+  }
+
+  scrollLoop = () => {
+    this.setPosition(window.scrollY * this.props.speed)
+    window.requestAnimationFrame(this.scrollLoop)
```

Comment:
No need to form an animation loop, because it would be inefficient to adjust the style on every frame even when there is no scrolling.

But setting position should be done on an animation frame.

```js
   updateParallax = () => {
      window.requestAnimationFrame(() => {
        this.setPosition(window.scrollY * this.props.speed)
      })
```